apps/models/app_types/custom/streamlit.py

```diff
@@ -0,0 +1,23 @@
+from ... import AppInstanceManager, BaseAppInstance
+from .base import AbstractCustomAppInstance
+
+
+class StreamlitAppInstanceManager(AppInstanceManager):
+    model_type = "streamlitappinstance"
+
+
+class StreamlitInstance(AbstractCustomAppInstance, BaseAppInstance):
+    objects = StreamlitAppInstanceManager()
+
+    def get_k8s_values(self):
+        k8s_values = super().get_k8s_values()
+        # TODO Change this to actual command to run gradio app
+        k8s_values["appconfig"][
+            "startupCommand"
+        ] = f"streamlit run /home/app.py --server.address=0.0.0.0 --server.port={self.port}"
```

Comment:
Similar to gradio I think this needs to be changed to 
```suggestion
        ] = f"streamlit run app.py --server.address=0.0.0.0 --server.port={self.port}"
```